src/h5web/vis-packs/core/shared/AxisSystem.tsx

```diff
@@ -64,17 +72,19 @@ function AxisSystem(props: Props) {
           type="abscissa"
           config={abscissaConfig}
           domain={xVisibleDomain}
-          canvasSize={size}
+          canvasSize={canvasSize}
         />
         <Axis
           type="ordinate"
           config={ordinateConfig}
           domain={yVisibleDomain}
-          canvasSize={size}
+          canvasSize={canvasSize}
           flipAxis
         />
       </Html>
-      <AxisSystemContext.Provider value={{ abscissaConfig, ordinateConfig }}>
+      <AxisSystemContext.Provider
+        value={{ visSize, abscissaConfig, ordinateConfig }}
```

Comment:
Providing `visSize` for use in `useCanvasScales`. Again, in my next PR, I'll replace all this with `{ abscissaScale, ordinateScale }`.